ci/Dockerfile.photon-3.0

```diff
@@ -0,0 +1,17 @@
+FROM photon:3.0
+
+MAINTAINER csiddharth@vmware.com
```

Comment:
Please change it to your name or remove it. Siddarth is not with VMware now

Reply:
Totally missed that. Fixed.